extensions/typescript-language-features/package.nls.json

```diff
@@ -39,6 +39,7 @@
 	"format.semicolons.ignore": "Don't insert or remove any semicolons.",
 	"format.semicolons.insert": "Insert semicolons at statement ends.",
 	"format.semicolons.remove": "Remove unnecessary semicolons.",
+	"format.indentSwitchCase": "Indent case clauses in switch statements.",
```

Comment:
@mjbvz should individual formatter preferences indicate what TS version they require? I’m realizing now I didn’t do that on `format.semicolons` back when I added that.

Reply:
Yes we should call this out. See some of the other setting strings for an example of this